src/main/java/org/thoughtcrime/securesms/ConversationListActivity.java

```diff
@@ -81,6 +81,7 @@ public class ConversationListActivity extends PassphraseRequiredActionBarActivit
   public static final String CLEAR_NOTIFICATIONS = "clear_notifications";
   public static final String ACCOUNT_ID_EXTRA = "account_id";
   public static final String FROM_WELCOME   = "from_welcome";
+  public static final String WARN_ADDR_EXTRA = "warn_addr";
```

Comment:
let's call it `WARN_CHATMAIL_ADDR_EXTRA` or `WARN_CANNOT_ENCRYPT` or otherwise be more explicit.